lib/verify/response.rb

```diff
@@ -75,20 +75,21 @@ def address_missing?
     end
 
     def full_name
-      first_name = most_recent_verified_value(attributes.fetch("firstNames"))
-      middle_name = most_recent_verified_value(attributes.fetch("middleNames"))
-      surname = most_recent_verified_value(attributes.fetch("surnames"))
+      first_name = most_recent_verified_value(attributes.fetch("firstNames"), required: true)
```

Comment:
if we're defaulting to `true` in the method signature I reckon these `require: true` calls are redundant.